### PR TITLE
DISC-46 Refactoring to Google coding standards (and addition of POLL_DELAY constant)

### PR DESCRIPTION
**Pull request by kfriedman**

### CODE REVIEW

File: src/main/java/org/nypl/harvester/sierra/utils/HarvesterConstants.java [hunk 1]
```diff
@@ -1,18 +1,20 @@
...
-	public static final String KINESIS_PARTITION_KEY = "CamelAwsKinesisPartitionKey";
-	public static final String KINESIS_SEQUENCE_NUMBER = "CamelAwsKinesisSequenceNumber";
-	public static final String APP_ITEMS_LIST = "items";
 
+  public static final String POLL_DELAY = "30s";
+  public static final String REDIS_KEY_LAST_UPDATED_TIME = "ItemUpdatePoller:lastUpdatedTime";
+  public static final String SIERRA_API_UPDATED_DATE = "updatedDate";
+  public static final String SIERRA_API_OFFSET = "offset";
+  public static final String SIERRA_API_LIMIT = "limit";
+  public static final String SIERRA_API_FIELDS_PARAMETER = "fields";
```
Comment (kfriedman) [inline src/main/java/org/nypl/harvester/sierra/utils/HarvesterConstants.java:10]: I added a `fields` constant here to use for the Sierra API call

Comment (jmathewt) [reply src/main/java/org/nypl/harvester/sierra/utils/HarvesterConstants.java:10]: Hey Kevin,

Are we using the fields query parameter in our sierra api call for the item id update poller right now?

Comment (kfriedman) [reply src/main/java/org/nypl/harvester/sierra/utils/HarvesterConstants.java:10]: Not right now. But, this is actually part of this PR though (changes to `ItemIdUpdateHarvester.java`).

**Comment by jmathewt** [comment]
@kfriedman will do the merge.

**Pull request closed without merging**